src/clr-angular/data/datagrid/index.ts

```diff
@@ -21,11 +21,14 @@ export * from './datagrid-placeholder';
 
 export * from './interfaces/state.interface';
 export * from './enums/sort-order.enum';
+export * from './enums/column-type.enum';
```

Comment:
I don't see the consumer using this enum in any of your examples, can we just keep it private?